src/components/Navbar/SettingsDrawer/TranslationSelectionBody.module.scss

```diff
@@ -24,3 +24,7 @@
 .translationGroup {
   margin-block-end: var(--spacing-large);
 }
+
```

Comment:
1. Use the same interface Tafsirs as well. 
2. Add search for both Qaris, and Tafsirs 
3. Update the fuzziness to be more strict, should only be tolerant to 1/2 letters